CHANGELOG.md

```diff
@@ -1,10 +1,17 @@
 # did-io ChangeLog
 
+## 0.9.0
```

Comment:
Is 0.9.0 needed if 0.8.0 was never released?

Reply:
Oh. Uh.. good point.